doc/develop/flash_debug/host-tools.rst

```diff
@@ -255,6 +255,73 @@ These debug host tools are compatible with the following debug probes:
 
 Check if your SoC is listed in `pyOCD Supported Devices`_.
 
+.. _lauterbach-trace32-debug-host-tools:
+
+Lauterbach TRACE32 Debug Host Tools
+***********************************
+
+`Lauterbach TRACE32`_ is a product line of microprocessor development tools,
+debuggers and real-time tracer with support for JTAG, SWD, NEXUS or ETM over
+multiple core architectures, including Arm Cortex-A/-R/-M, RISC-V, Xtensa, etc.
+Zephyr allows users to develop and program boards with Lauterbach TRACE32
+support using :ref:`west <west-flashing>`.
+
+The runner consists of a wrapper around TRACE32 software, and allows a Zephyr
+board to execute a custom start-up script (Practice Script) for the different
+commands supported, including the ability to pass extra arguments from CMake.
+Is up to the board using this runner to define the actions performed on each
+command.
+
+Install Lauterbach TRACE32 Software
+-----------------------------------
+
+Download Lauterbach TRACE32 software from the `Lauterbach TRACE32 download website`_
+(registration required) and follow the installation steps described in
+`Lauterbach TRACE32 Installation Guide`_.
+
+Flashing and Debugging
+----------------------
+
+Set the environment variable :envvar:`T32_DIR` pointing to the TRACE32 system
```

Comment:
Please add a `:ref:` to env_vars.rst instead of restating parts of that page. For instance:

```
Set the :ref:`environment variable <env_vars>` :envvar:`T32_DIR` to ...
```

Reply:
do you suggest to remove the example of setting the envvar and just keep the text above instead?

Reply:
>  Do you suggest to also remove the example of setting the envvar and just keep the text above instead?

Yes, please.

Reply:
removed